cf/bioinformatics-access-group.yaml

```diff
@@ -35,7 +35,9 @@ Resources:
                   - 's3:PutObject'
                 Resource:
                   - !Sub "arn:aws:s3:::biomage-originals-${Environment}/*"
+                  - !Sub "arn:aws:s3:::biomage-originals-${Environment}-${AWS::AccountId}/*"
                   - !Sub "arn:aws:s3:::biomage-source-${Environment}/*"
+                  - !Sub "arn:aws:s3:::biomage-source-${Environment}-${AWS::AccountId}/*"
```

Comment:
Are these policies a duplicate of the s3 policies in the developer-access-group?

Reply:
No, these changes are made so that it's up to date with the master branch for this file.